required/latex-lab/testfiles-OR/footmisc-001.tlg

```diff
@@ -60,4 +60,7 @@ Completed box being shipped out [1]
 ....\glue 0.0 plus 1.0fil
 ..\pdfrunninglinkon
 .\kern 0.0
+.\kern -633.0
+.\hbox(0.0+0.0)x0.0
+.\kern 633.0
```

Comment:
where is that coming from? just curious

Reply:
The xmp-code uses the shipout/lastpage hook (for dvi based engine it must be in a box). And as soon as this hook is not empty you get this box:
~~~~
\documentclass{article}
\AddToHook{shipout/lastpage}{\typeout{something}}
\begin{document}
blub\showoutput
\end{document}
~~~~

I will correct the rest.